### PR TITLE
Update mock_jsons/

### PR DESCRIPTION
**Pull request by mlangsdorf**
Add a new tests/mock_jsons/3.9.41/ directory with mock JSON data taken from recent runs of osbs-client.

### CODE REVIEW

**Comment by twaugh** [comment]
1.0.6? As in OpenShift 1.0.6?
Is that the version you ran these queries against, or was it newer?

**Comment by mlangsdorf** [comment]
Okay, this is now 3.9.41.

**Comment by mlangsdorf** [comment]
Removed all instances of redhat.com and replaced them with an example.com

File: tests/fake_api.py [hunk 1]
```diff
@@ -498,7 +498,7 @@ def delete(self, url, *args, **kwargs):
         return self.request(url, "delete", *args, **kwargs)
 
 
-@pytest.fixture(params=["1.0.4"])
+@pytest.fixture(params=["1.0.4", "3.9.41"])
```
Comment (MartinBasti) [inline tests/fake_api.py]: Do we still care about version `1.0.4`?

**Comment by mlangsdorf** [comment]
Not updating release notes because this is a change to our internal test infrastructure.

I'd like to wait a week or 3 for these values to go through test before removing 1.0.4.

**Comment by MartinBasti** [comment]
Ok, thanks

**Pull request merged**